tests/blockchain/regtest/SimpleTransfer.cpp

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2010-2022 The Open-Transactions developers
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include <gtest/gtest.h>
+#include <chrono>
+#include <thread>
+
+#include "opentxs/OT.hpp"
+#include "opentxs/api/Context.hpp"
+#include "opentxs/blockchain/crypto/Subchain.hpp"
+#include "opentxs/util/Log.hpp"
+#include "ottest/fixtures/blockchain/Regtest.hpp"
+#include "ottest/fixtures/blockchain/RegtestSimple.hpp"
+#include "ottest/fixtures/paymentcode/VectorsV3.hpp"
+
+namespace ottest
+{
+
+TEST_F(Regtest_fixture_simple, send_to_client)
+{
+    EXPECT_TRUE(Start());
+    EXPECT_TRUE(Connect());
+
+    const std::string name_alice = "Alice";
+    const std::string name_bob = "Bob";
+
+    Height targetHeight = 0, begin = 0;
```

Comment:
To be consistent in variable naming convention I suggest to rename it to: target_height :)